static/scripts/util.js

```diff
@@ -105,4 +105,15 @@ $(document).on('click', '.remove-tag', function (e) {
     $(this).parent('.badge').remove();
 });
 
+$(document).on('click', '#filterIcon', function (e) {
+    e.preventDefault();
+    $('#filterRow').toggle();
+});
 
+$(document).on('keyup', '#filterRow input', function () {
+    let column = $(this).data('column');
+    let value = $(this).val().toLowerCase();
+    $('table tbody tr').filter(function () {
```

Comment:
```suggestion
    $('table tbody tr').each(function () {
```

as you don't seem to use the filtered list.